src/Raven.Studio/wwwroot/App/views/shell/registration.html

```diff
@@ -46,15 +46,25 @@ <h3 class="modal-title" id="myModalLabel">
             </div>
             <div class="modal-footer">
 
-
                 <div class="btn-group" data-bind="visible: dismissVisible()">
                     <button class="btn btn-default" data-bind="click: $root.dismiss.bind($root, 1)">
                         <i class="icon-recent"></i>
                         <span>Dismiss for 1 day</span>
                     </button>
                 </div>
 
-                <button type="submit" data-bind="css: { 'btn-spinner': isBusy }, enable: !isBusy()" class="btn btn-primary">Submit</button>
+                <!-- ko if: canForceUpdate -->
+                <button class="btn btn-info" data-bind="click: forceLicenseUpdate, disable: spinners.forceLicenseUpdate,
+                                    css: { 'btn-spinner': spinners.forceLicenseUpdate }">
+                    <i class="icon-force"></i>
+                    <span>Force Update</span>
+                </button>
+                <!-- /ko -->
+
+                <button type="submit" data-bind="css: { 'btn-spinner': isBusy }, enable: !isBusy(), disable: spinners.forceLicenseUpdate"
```

Comment:
you can't use both enable and disable bindings together.

Reply:
done